required/latex-lab/latex-lab-table.dtx

```diff
@@ -1425,9 +1425,9 @@
       }
     \hbox to\z@{\hss\parbox[t]\LTcapwidth{%
     \reset@font
-    \tag_stop:n{caption}
+    \tag_suspend:n{caption}
     \sbox\@tempboxa{#1{#2:~}#3}%
-    \tag_start:n{caption}
+    \tag_resume:n{caption}
```

Comment:
yes, if you save a box and then reuse it directly then you want its internal tagging to be preserved. But it could be useful to have an `\untaggedsbox` command.

Reply:
I was thinking the other way around, make sbox untagged by default and have a \taggedsbox, but it kind of depends what happens more often.